compiler/src/org.graalvm.compiler.asm.amd64/src/org/graalvm/compiler/asm/amd64/AVXKind.java

```diff
@@ -41,26 +41,36 @@
 public final class AVXKind {
 
     public enum AVXSize {
-        DWORD,
-        QWORD,
-        XMM,
-        YMM,
-        ZMM;
-
-        public int getBytes() {
-            switch (this) {
-                case DWORD:
-                    return 4;
-                case QWORD:
-                    return 8;
-                case XMM:
-                    return 16;
-                case YMM:
-                    return 32;
-                case ZMM:
-                    return 64;
+        DWORD(4),
+        QWORD(8),
+        XMM(16),
```

Comment:
Are we sure that this isn't somewhere else in JVMCI/Graal? There are constants used for the AMD64 registers in various places (`xmm0`, etc) - is it worth repeating this?